src/Sql/dbo/Stored Procedures/User_Update.sql

```diff
@@ -29,7 +29,8 @@
     @KdfIterations INT,
     @CreationDate DATETIME2(7),
     @RevisionDate DATETIME2(7),
-    @ApiKey VARCHAR(30)
+    @ApiKey VARCHAR(30),
+    @ForcePasswordReset BIT
```

Comment:
Same here, my recommendation would be to start setting default parameters for these sprocs.

```suggestion
    @ForcePasswordReset BIT = 0
```

Reply:
Updated